examples/chart/teleport-cluster/values.yaml

```diff
@@ -363,6 +365,37 @@ gcp:
   # You can override this to a blank value if the worker node running Teleport already has a service account which grants access.
   credentialSecretName: teleport-gcp-credentials
 
+#####################################################
+# Azure-specific settings (only used in "azure" mode)
+#####################################################
+azure:
+  # The Postgres database hostname hosted in Azure. It must not contain the
+  # suffix ".postgres.database.azure.com".
+  databaseHost: ""
+  # The Postgres user Teleport must use to connect to the backend and audit
+  # databases.
+  databaseUser: ""
+  # The Postgres Database to use for backend storage.
+  backendDatabase: ""
+  # The Postgres Database to use for audit log storage.
+  # This MUST NOT be the same database as used for 'backendDatabase'.
+  auditLogDatabase: ""
+  # Whether to mirror audit log entries to stdout in JSON format (useful for external log collectors)
+  auditLogMirrorOnStdout: false
+  # The Google storage bucket name to use for recorded session storage. This bucket must already exist in the Google account being used.
+  sessionRecordingStorageAccount: ""
+  # Azure client ID is used by the Kubernetes Service Account to know which
+  # Application it should impersonate. This can be unset only if the clientID is
+  # passed through other means (e.g. environment variable)
+  clientID: ""
+  # Controls the `pool_max_conns` setting passed to PostgreSQL. This is the
+  # max amount of connections Teleport can open to the database. This can affect
+  # performance on large clusters and depends on various factors like the
+  # database size, the number of CPU cores available for Teleport, GOMAXPROCS
+  # and the database latency.
+  # This only applies to the core backend connections, not the audit log ones.
+  # 0 means the parameter is not set and the client's default is used (recommended)
+  databasePoolMaxConnections: 0
```

Comment:
I think leaving the default is saner, it will detect the maximum amount of CPUs it can run on. This doesn't means it won't get throttled after 5ms, but it makes sense from a concurrency pov.